File: dataBase（数据库）/0SaveUploadFile/bubble_sort1.py
# -*- coding: utf-8 -*-
# 冒泡排序
def  bubbleSort(numbers):
	# 冒泡算法的实现
	for j in range(len(numbers)-1,-1,-1):

		for i in range(j):
			if numbers[i] > numbers[i+1]:
				#把数字大的放到后面
				numbers[i], numbers[i+1] = numbers[i+1], numbers[i]
			print(numbers)
			

File: dataBase（数据库）/Sorting/BubbleSort1.java


public class BubbleSort {
	public static void bubbleSort(int[] a){
		for(int i=0;i<a.length-1;i++){
			for(int j=1;j<a.length-i;j++){
				int temp;
				if(a[j]<a[j-1]){
					temp=a[j];
					a[j]=a[j-1];
					a[j-1]=temp;
				}
			}
		}
	}

}
